tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -264,6 +271,59 @@ RHODS Dahsboard Pod Should Contain OauthProxy Container
         List Should Contain Value    ${container_name}    oauth-proxy
     END
 
+Is Environment Staging
+    [Documentation]    Return true if environment is staging otherwise fails
+    ${list_of_links} =    Get Links From Switcher
+    Should Contain    ${list_of_links}[0]   devshift.org
+
+Check Application Switcher Links To Openshift Cluster Manager
+    ${cluster_id} =    Get Cluster ID
+    ${cluster_id} =    Remove String    ${cluster_id}    "
+    ${ocm_staging_link} =    Set Variable    https://qaprodauth.cloud.redhat.com/openshift/details/${cluster_id}
+    ${ocm_prod_link} =    Set Variable    https://cloud.redhat.com/openshift/details/${cluster_id}
```

Comment:
you should use the link from the switcher for the production

Reply:
Done